server/index.js

```diff
@@ -1,11 +1,36 @@
-const express = require('express')
-const app = express()
-const port = 5000
+
+const express = require('express');
+const mongoose = require('mongoose')
+
+const UserRoute = require('./src/routes/UserRoute');
+const config = require('./src/config/config')
+const app = express();
+app.use(express.json())
+
+const apiRouter = express.Router()
+app.use("/api", apiRouter)
+
+const userRoute = new UserRoute()
+userRoute.initRoutes(apiRouter)
 
 app.get('/', (req, res) => {
   res.send('Hello, world!')
 })
 
-app.listen(port, () => {
-  console.log(`Server is running on port ${port}`)
-})
+const connectDB = async () => {
+  try {
+    console.log(config.mongo.uri)
+    await mongoose.connect(config.mongo.uri)
+    startServer()
+  } catch (err) {
+    console.error(err)
+  }
+}
+
+const startServer = () => {
+  app.listen(config.server.port, () => {
+    console.log(`Server is running on port ${config.server.port}`);
```

Comment:
@NLmeng was the linting  rule not enforced over here for the semi-colon?

Reply:
Sorry yea, I wrote a sticky note to run the esllint stuff...I merged dev after I created this PR.

Reply:
i think you deleted some necessary dependencies when fixing conflicts. if nothing is wrong, then when you commit, it should auto lint and format everything